src/main.py

```diff
@@ -87,14 +87,22 @@ def setUp(self):
         # make sure we show the UI before run completes, as otherwise
         # it is possible the run can complete before the UI is shown,
         # causing the app to shut down early
-        self.webview.show()
+        self.view.show()
         return 0
 
     def load_complete(self):
         """
-        This is a PyEverywhere delegate method to let us know the WebView is ready to use. Just pass for now.
+        This is a PyEverywhere delegate method to let us know the WebView is ready to use.
         """
-        pass
+
+        # On Android, there is a system back button, that works like the browser back button. Make sure we clear the
+        # history after first load so that the user cannot go back to the loading screen. We cannot clear the history
+        # during load, so we do it here.
+        # For more info, see: https://stackoverflow.com/questions/8103532/how-to-clear-webview-history-in-android
+        if pew.ui.platform == 'android':
+            # FIXME: Change pew to reference the native webview as webview.native_webview rather than webview.webview
+            # for clarity.
+            self.view.webview.webview.clearHistory()
```

Comment:
Wouldn't doing this here interfere with the stuff on [line 130](https://github.com/learningequality/kolibri-installer-android/pull/31/files#diff-0c81b9c94c68ea34c497c1fa52daee56R130)? I haven't tested, but I assume we'd want a user to be able to go back on restore?

Reply:
Even though we call it restore, it is the app restarting. It's just remembering the state at the time it was killed. So the Kolibri server startup process will run on restore, and the loading page will still show. (This is why we don't show it conditional on a clean start.)